From 661b4c80512dc41353b0f3639c512d719cbf9d86 Mon Sep 17 00:00:00 2001
From: Dennis Gilmore <dennis@ausil.us>
Date: Thu, 6 Jun 2013 04:19:38 -0500
Subject: [PATCH] update extlinux.conf on arm arches if it exists

---
 new-kernel-pkg | 1 +
 1 file changed, 1 insertion(+)

diff --git a/new-kernel-pkg b/new-kernel-pkg
index f248e7e..3206155 100755
--- a/new-kernel-pkg
+++ b/new-kernel-pkg
@@ -82,6 +82,7 @@ elif [ $ARCH = 's390' -o $ARCH = 's390x' ]; then
 elif [[ ${ARCH} =~ armv[5|7].*l ]] ; then
     liloConfig=""
     bootPrefix=/boot
+    extlinuxConfig=$(readlink -f /etc/extlinux.conf 2>/dev/null)
     ubootDir=${UBOOT_DIR:-"/boot"}
     ubootScript=$ubootDir/${UBOOT_SCR:-"boot.scr"}
     ubootKList=${UBOOT_KLIST:-"klist.txt"}
-- 
1.8.2.1

